opentelemetry/server/src/main/kotlin/opentelemetry/ktor/example/plugins/routing/Routing.kt

```diff
@@ -0,0 +1,106 @@
+package opentelemetry.ktor.example.plugins.routing
+
+import opentelemetry.ktor.example.CUSTOM_HEADER
+import opentelemetry.ktor.example.CUSTOM_METHOD
+import opentelemetry.ktor.example.CUSTOM_METHOD_NOT_KNOWN
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.server.websocket.*
+import io.ktor.websocket.*
+import io.opentelemetry.api.trace.Span
+import kotlinx.coroutines.delay
+import opentelemetry.ktor.example.plugins.opentelemetry.installOpenTelemetryOnServer
+import opentelemetry.ktor.example.plugins.opentelemetry.serviceName
+
+fun Application.configureRouting() {
+    install(WebSockets)
+
+    val openTelemetry = installOpenTelemetryOnServer()
```

Comment:
Can it be an extension on application (stored in the attributes)?

Reply:
Thanks for the review! I'm not sure about an extension on the application, because users should provide an instance of `OpenTelemetry` by themselves (in our example we do it in `fun getOpenTelemetry(...)` from `utils.kt`).  But please correct me if I misunderstood you